### PR TITLE
Updates Check Deposit Text / Docs

### PR DESCRIPTION
**Pull request by jaspermayone**
## Summary of the problem

Closes #5352 

## Describe your changes

This changes the "how do i..." from sending a check to reimbursements, and the FAQ test to @dwahbe's text from #5352. 

CleanShot 2024-04-25 at 01 58 07@2x
CleanShot 2024-04-25 at 01 59 14@2x

CleanShot 2024-04-25 at 01 59 55@2x

### CODE REVIEW

**Comment by jaspermayone** [comment]
Does it make sense to wait to merge this for docs from @sampoder and/or @YodaLightsabr?

File: app/views/events/_faq.html.erb [hunk 1]
```diff
@@ -133,44 +133,52 @@
...
-          <h3 class="inline mb1">I need to deposit a paper check. What should I do?</h3>
-        </summary>
-        <article>
-          <p>
-            HCB accepts paper checks! Just mail the check to our
-            billing address:
-          </p>
-          <code>
+   <li class="mb3" id="check">
+  <details open>
```
Comment (garyhtou) [inline app/views/events/_faq.html.erb]: can you fix the indent here? something looks funky lol

File: app/views/events/_faq.html.erb [hunk 2]
```diff
@@ -133,44 +133,52 @@
...
-            check's memo or somewhere else in or on the envelope sent.
-            Without it, we can't properly route received funds.
-          </p>
 
-          <p>
-            <strong>Checks must be made out to either "The Hack Foundation"
-            or "Hack Club".</strong> Our banking partners will, unfortunately,
-            reject checks made out to any name besides those two.
-          </p>
+      <.>
```
Comment (garyhtou) [inline app/views/events/_faq.html.erb]: Should this be a `<p>` tag?

File: app/views/events/documentation.html.erb [hunk 3]
```diff
@@ -41,7 +41,7 @@
 <hr class="my3">
 <h3>How do I...</h3>
 <ul class="grid mb3">
-  <%= link_to( event_receive_check_path ) do %>
+  <%= link_to( event_send_reimbursement_path ) do %>
```
Comment (garyhtou) [inline app/views/events/documentation.html.erb]: To answer your question, let's not wait on Sam's copy — I think that one might take a while to finish up. Instead, let's keep this placeholder, but disable it (make the card non-clickable) with the message "coming soon"

File: app/views/events/_faq.html.erb (hunk 2 above)
Comment (jaspermayone) [reply app/views/events/_faq.html.erb]: yes -_- fixing rn

**Comment by jaspermayone** [comment]
@garyhtou do you want me to see if I can resolve these merge conflicts?

**Comment by garyhtou** [comment]
Yes, please! feel free to ping us if you need help

**Comment by jaspermayone** [comment]
@garyhtou merge conflicts have been resolved, and this is ready for review!

**Review by sampoder — DISMISSED** [review]
LGTM! Thanks for doing this.

**Pull request merged**